.github/workflows/image-scan.yml

```
@@ -0,0 +1,202 @@
+          fi
+        done
+
+    - name: Build PR comment
+      if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize')
+      id: build-pr-comment
+      shell: bash
+      run: |
+        lwResultFiles="$(find . -type f -name '*.lacework-result.json' | sort)"
+        
+        # Collect overall status.
+        lwStatusIcon=":green_circle:"
+        for jsonFile in $lwResultFiles; do
+          cveC=$(jq -r '.stats.Critical.total' $jsonFile)
+          cveH=$(jq -r '.stats.High.total' $jsonFile)
+          if [ $cveC -gt 0 ] || [ $cveH -gt 0 ]; then
+            lwStatusIcon=":red_circle:"
+            break
+          fi
+        done
+
+        c="### ${lwStatusIcon}&nbsp; Lacework scan results:"
+        c="${c}\n| Image | Critical | High | Medium | Low | Info |"
+        c="${c}\n|---|---|---|---|---|---|"
+
+        # Highlight number in table cell.
+        _hln() {
+          if [ ${1} -gt 0 ]; then
+            echo "**${1}** :x:"
+          else
+            echo "${1} :heavy_check_mark:"
+          fi
+        }
+
+        for jsonFile in $lwResultFiles; do
+          image=$(jq -r '.image' $jsonFile)
+          cveC=$(jq -r '.stats.Critical.total' $jsonFile)
+          cveH=$(jq -r '.stats.High.total' $jsonFile)
+          cveM=$(jq -r '.stats.Medium.total' $jsonFile)
+          cveL=$(jq -r '.stats.Low.total' $jsonFile)
+          cveI=$(jq -r '.stats.Info.total' $jsonFile)
+          c="${c}\n| ${image} | $(_hln $cveC) | $(_hln $cveH) | $cveM | $cveL | $cveI |"
+        done
+
+        runUrl="${{ github.server_url }}/${{ github.repository }}/actions/runs/${{ github.run_id }}#user-content-scan-results"
+        c="${c}\n\n<sup>[Go to job summary for more details...]($runUrl)</sup>"
+
+        # Set multiline output.
+        echo 'comment<<EOF' >> $GITHUB_OUTPUT
+        echo -e "$c" >> $GITHUB_OUTPUT
+        echo 'EOF' >> $GITHUB_OUTPUT
+
+    - name: Add PR comment (if content changed)
+      if: github.event_name == 'pull_request' && (github.event.action == 'opened' || github.event.action == 'synchronize')
+      uses: actions/github-script@v6
+      with:
+        retries: 3
+        retry-exempt-status-codes: 400,401
+        script: |
+          function removeLinks(body) {
+            return body.replace(/\]\(http[^)]+\)/gm, "](#)");
+          }
+
+          // Get the existing comments.
+          const {data: comments} = await github.rest.issues.listComments({
+            owner: context.repo.owner,
+            repo: context.repo.repo,
+            issue_number: context.payload.number,
+          })
+
+          // Find last bot comment.
+          const lastBotComment = comments.slice().reverse().find(comment => comment.user.login === 'github-actions[bot]')
+
+          const newCommentBody = `${{ steps.build-pr-comment.outputs.comment }}`
+          if (!lastBotComment || removeLinks(newCommentBody)  !== removeLinks(lastBotComment.body)) {
```

super-ultra-OCD-nit 😅 
```suggestion
          if (!lastBotComment || removeLinks(newCommentBody) !== removeLinks(lastBotComment.body)) {
```